spec.html

```diff
@@ -13610,14 +13610,26 @@ <h1>
+    <p>This specification defines many built-in function objects. The effect of calling such a function must conform to its specification. An implementation may also provide additional built-in function objects that are not defined in this specification.</p>
+    <p>A built-in function object must have the ordinary object behaviour and internal slots specified in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>. In addition, it must have the following internal slots:</p>
+    <ul>
+      <li>[[Realm]], a Realm Record that represents the realm in which the function was created.</li>
+      <li>[[InitialName]], a String value that is the initial name of the function. It is used by <emu-xref href="#sec-function.prototype.tostring"></emu-xref>.</li>
+    </ul>
     <p>Unless otherwise specified every built-in function object has the %Function.prototype% object as the initial value of its [[Prototype]] internal slot.</p>
-    <p>The behaviour specified for each built-in function via algorithm steps or other means is the specification of the function body behaviour for both [[Call]] and [[Construct]] invocations of the function. However, [[Construct]] invocation is not supported by all built-in functions. For each built-in function, when invoked with [[Call]], the [[Call]] _thisArgument_ provides the *this* value, the [[Call]] _argumentsList_ provides the named parameters, and the NewTarget value is *undefined*. When invoked with [[Construct]], the *this* value is uninitialized, the [[Construct]] _argumentsList_ provides the named parameters, and the [[Construct]] _newTarget_ parameter provides the NewTarget value. If the built-in function is implemented as an ECMAScript function object then this specified behaviour must be implemented by the ECMAScript code that is the body of the function. Built-in functions that are ECMAScript function objects must be strict functions. If a built-in constructor has any [[Call]] behaviour other than throwing a *TypeError* exception, an ECMAScript implementation of the function must be done in a manner that does not cause the function's [[IsClassConstructor]] internal slot to have the value *true*.</p>
+    <p>The behaviour specified for each built-in function via algorithm steps or other means is the specification of the function body behaviour for both [[Call]] and [[Construct]] invocations of the function. However, [[Construct]] invocation is not supported by all built-in functions. For each built-in function, when invoked with [[Call]], the [[Call]] _thisArgument_ provides the *this* value, the [[Call]] _argumentsList_ provides the named parameters, and the NewTarget value is *undefined*. When invoked with [[Construct]], the *this* value is uninitialized, the [[Construct]] _argumentsList_ provides the named parameters, and the [[Construct]] _newTarget_ parameter provides the NewTarget value.</p>
```

Comment:
I don't think this paragraph is needed. It duplicates what is already described in step 10 of `[[Call]]` and `[[Construct]]`. The only thing that's possibly useful is "[[Construct]] invocation is not supported by all built-in functions", but even then I wouldn't find that useful unless we were more explicit about what it meant to be "not supported".